themes/cclw/constants/faqs.tsx

```diff
@@ -261,11 +255,7 @@ export const FAQS: TFAQ[] = [
     content: (
       <>
         <p>
-          You’ll need to{" "}
-          <ExternalLink url="https://docs.google.com/forms/d/e/1FAIpQLSdFkgTNfzms7PCpfIY3d2xGDP5bYXx8T2-2rAk_BOmHMXvCoA/viewform">
-            fill out this form
-          </ExternalLink>{" "}
-          to request our entire dataset.
+          You’ll need to <ExternalLink url="https://form.jotform.com/233131638610347">fill out this form</ExternalLink> to request our entire dataset.
```

Comment:
Why as the {" "} been removed?

Reply:
It's not needed, the whitespace is still there. The line breaking previously meant that we needed to add an explicit {" "} otherwise the space character would be trimmed as it was at the end of a line to prevent trailing whitespace -- that is no longer needed.

Reply:
![image](https://github.com/climatepolicyradar/navigator-frontend/assets/46493669/7473d0da-8e6c-4c9c-a3e7-181cdb7b9cab)